### PR TITLE
chore: remove metric from performance

### PR DESCRIPTION
**Pull request by ThibaultFy**
## Description

Remove metric from the performance data model

## Companion PR

- https://github.com/Substra/substra-backend/pull/650 
- https://github.com/Substra/orchestrator/pull/213 (main)
- https://github.com/Substra/substra/pull/358

### CODE REVIEW

File: backend/api/tests/views/test_views_performance.py [hunk 1]
```diff
@@ -376,5 +358,5 @@ def test_n_plus_one_queries_performance_list(authenticated_client, create_comput
     with utils.CaptureQueriesContext(connection) as query:
         print(authenticated_client.get(url))
     query_task_with_perf = len(query.captured_queries)
-    assert query_task_with_perf < 12
-    assert query_task_with_perf - query_tasks_empty < 4
+    assert query_task_with_perf < 11
```
Comment (SdgJlbl) [inline backend/api/tests/views/test_views_performance.py:361]: 🎉

File: backend/orchestrator/common_pb2.pyi [hunk 2]
```diff
@@ -68,6 +68,7 @@ ASCENDING: SortOrder.ValueType  # 1
 DESCENDING: SortOrder.ValueType  # 2
 global___SortOrder = SortOrder
 
+@typing_extensions.final
```
Comment (SdgJlbl) [inline backend/orchestrator/common_pb2.pyi:71]: Out of curiosity, what is this line doing?

Comment (SdgJlbl) [reply backend/orchestrator/common_pb2.pyi:71]: Oh, nice https://peps.python.org/pep-0591/#the-final-decorator

Comment (ThibaultFy) [reply backend/orchestrator/common_pb2.pyi:71]: It was auto generated, I didn't have your curiosity ahah. Thanks for pointing that out, really interesting indeed !

**Pull request merged**